emails/loader/helpers.py

```diff
@@ -4,7 +4,7 @@
 
 import re
 import cgi
-import charade
+import chardet
```

Comment:
Although you updated ```install_requires```, I suppose we should provide some fallback here, like this:

    try:
        import charade as chardet
        warnings.warn("charade module is deprecated, update your requirements to chardet", DeprecationWarning)
    except ImportError:
        import chardet